docontrol/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "7622c46e-bade-44ce-99e1-a3529bf4fc04",
+  "app_id": "docontrol",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "SaaS Data Security - modernizing DLP and CASB to Secure SaaS Data",
+    "title": "DoControl",
+    "media": []
+  },
+  "author": {
+    "homepage": "https://www.docontrol.io/",
+    "name": "DoControl",
+    "support_email": "support@docontrol.io",
+    "sales_email": "sales@docontrol.io"
+  },
+  "oauth": {},
+  "assets": {
+    "integration": {
+      "source_type_name": "DoControl",
+      "configuration": {},
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "docontrol.",
+        "check": "",
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    }
```

Comment:
Since you are not directly submitting any data into Datadog or pulling any data out, this section can be removed, and the `metadata.csv` and `service_checks.json` files can be deleted.

Reply:
Which section should be deleted? Assets or Integration or metrics and service_checks?

Reply:
The `integration` section can be removed, leaving `assets` as `{ }`